Source/track.cpp

```diff
@@ -2,75 +2,49 @@
 
 #include "../types.h"
 
-bool sgbIsScrolling; // weak
-int track_cpp_init_value; // weak
-int sgdwLastWalk; // weak
-bool sgbIsWalking; // weak
-
-const int track_inf = 0x7F800000; // weak
-
-struct track_cpp_init

```

Comment:
Since you know how to deal with this bit could you make a PR for removing them from all files where relevant?

Reply:
The proper way to fix it is to put the INFINITY assignment in a header and then include that header in all the affected source files. But at the moment we're just including every header in every source so there's not really a right way to do it